web/app/js/components/Octopus.jsx

```diff
@@ -72,6 +107,19 @@ export default class Octopus extends React.Component {
     );
   }
 
+  renderCollapsedNeighbors = neighbors => {
+    return (
+      <div key="unmeshed-resources" className="octopus-body neighbor collapsed">
+        {
+          _.map(neighbors, r => {
+            let display = displayName(r);
+            return <div className="octopus-title neighbor-title">{this.linkedResourceTitle(r, display)}</div>;
```

Comment:
I get this error in the console:
<img width="849" alt="screen shot 2018-09-28 at 10 31 46 am" src="https://user-images.githubusercontent.com/2197104/46224057-08d3fd00-c30a-11e8-8dc3-c236cd531354.png">

It looks like the line `<div className="octopus-title neighbor-title">{this.linkedResourceTitle(r, display)}</div>` needs a key field for React to not complain.

Reply:
ah yeah! good catch!